pay-api/src/pay_api/schemas/schemas/payment_info.json

```diff
@@ -23,6 +23,31 @@
         "PAYBC"
       ],
       "pattern": "^(.*)$"
+    },
+    "transitNumber": {
```

Comment:
do we need to prefix with bank as well? like bankTransitNumber ; bankAccountNumber

Reply:
Yes, I have that in model but not in schema. 👍